### PR TITLE
#167313368 User should be able to clap for and dislike an article

### PR DESCRIPTION
**Pull request by minega25**
#### What does this PR do?
 User should be able to clap for and dislike a comment.
#### Description of Task to be completed?
I want to be able to clap or dislike articles, both authored by me and those not authored by me.
#### How should this be manually tested?
Visit single article view.
User should be able to dislike or clap for that particular article
#### What are the relevant pivotal tracker stories?
[#167313368](https://www.pivotaltracker.com/story/show/167313368)
Screen Shot 2019-11-03 at 20 17 49
Screen Shot 2019-11-03 at 20 16 58

### CODE REVIEW

File: src/components/common/articleCard/articleCardTextComponent/index.js [hunk 1]
```diff
@@ -15,10 +16,12 @@ const ArticleCardTextComponent = ({ title, description, views }) => (
       <div className="landingArticleCardText__views">
         <p>
           <span>{views}</span>
-          {' '}
 views
         </p>
       </div>
+      <div>
+        <Like />
+      </div>
```
Comment (Anguandia) [inline src/components/common/articleCard/articleCardTextComponent/index.js]: Would you please throw more light on the hard-coded likes' value of 100?

**Review by Anguandia — COMMENTED** [review]
Looks Good!

File: src/components/common/articleCard/articleCardTextComponent/index.js (hunk 1 above)
Comment (minega25) [reply src/components/common/articleCard/articleCardTextComponent/index.js]: Worked on this.

**Review by salviosage — APPROVED** [review]
LGTM

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]
Good job. Though the UI would look better if the clap and the dislike button were the same sizes and in the same horizontal line as the social media icons.

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]
The clap and dislike buttons should not appear if you are the author of the article

**Comment by minega25** [comment]
> The clap and dislike buttons should not appear if you are the author of the article

@SokoPaulSokool I have worked on both feedback.

**Review by habinezadalvan — APPROVED** [review]
LGTM

**Pull request merged**